js/components/SyriacLetter.js

```diff
@@ -0,0 +1,109 @@
+    { id: 19, letter: "Kaph", is_script: true, display: "ܟ", script: "estrangela", leading_letter: "ܝ" },
+    { id: 20, letter: "Kaph (Final)", is_script: true, display: "ܟ", script: "estrangela" },
+    { id: 25, letter: "Lamadh (Final, open)", is_script: true, display: "ܠ", script: "estrangela" },
+    { id: 21, letter: "Lamadh", is_script: true, display: "ܠ", script: "estrangela", trailing_letter: "ܝ" },
+    { id: 23, letter: "Lamadh (Final, closed)", is_script: true, display: "ܠ", script: "serto" },
+    { id: 26, letter: "Mim", is_script: true, display: "ܡ", script: "estrangela", leading_letter: "ܝ" },
+    { id: 27, letter: "Mim (Final)", is_script: true, display: "ܡ", script: "estrangela"},
+    { id: 28, letter: "Nun", is_script: true, display: "ܢ", script: "estrangela", leading_letter: "ܝ" },
+    { id: 30, letter: "Nun (Final, connected)", is_script: true, display: "ܢ", script: "estrangela", trailing_letter: "ܝ" },
+    { id: 31, letter: "Nun (Final, unconnected)", is_script: true, display: "ܢ", script: "estrangela" },
+    { id: 38, letter: "Semkath", is_script: true, display: "ܣ", script: "estrangela" },
+    { id: 5, letter: "Ayin", is_script: true, display: "ܥ", script: "estrangela" },
+    { id: 32, letter: "Pe", is_script: true, display: "ܦ", script: "estrangela" },
+    { id: 37, letter: "Sadhe", is_script: true, display: "ܨ", script: "estrangela" },
+    { id: 33, letter: "Qaph", is_script: true, display: "ܩ", script: "estrangela" },
+    { id: 35, letter: "Rish (Angular)", is_script: true, display: "ܪ", script: "estrangela" },
+    { id: 36, letter: "Rish (Round)", is_script: true, display: "ܪ", script: "serto" },
+    { id: 39, letter: "Shin", is_script: true, display: "ܫ", script: "estrangela" },
+    { id: 43, letter: "Taw (Looped)", is_script: true, display: "ܬ", script: "estrangela" },
+    { id: 44, letter: "Taw (Triangular)", is_script: true, display: "ܬ", script: "serto", trailing_letter: "ܝ" },
+    { id: 42, letter: "Taw (L-shaped)", is_script: true, display: "ܬ", script: "serto" }
+];
+
+
+class SyriacLetter extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.letter = letters.find(lt => lt.id == this.props.id);
+    this.trailing = "";
+    this.leading = "";
+
+    if (this.letter !== undefined) {
+      this.display = this.letter.display;
+      this.script = this.letter.script;
+      /* Fonts provided by https://sedra.bethmardutho.org/about/fonts */
+      if (this.script == "serto") {
+        this.font = "SertoJerusalem";
+      } else if (this.script == "estrangela") {
+        this.font = "EstrangeloEdessa";
+      } else {
+        this.font = "sans-serif";
+      }
+      /* Prefix final forms (and wrap medial forms) via transparent
+       * "spacer" letters to make them display properly. Note that
+       * positions are read from right to left. 
+       * XXX This trick may only work in Firefox :-(
```

Comment:
Have you confirmed whether it only works in Firefox?

Reply:
Well it doesn't work with Chrome or Safari on my laptop -- and on my phones, for what that's worth. Too bad! Maybe there's some other trick that would trick those browsers, but otherwise, I don't know, maybe we'd need to add a browser check and show canned letter images instead.